pkg/crc/machine/bundle/repository.go

```diff
@@ -127,12 +127,17 @@ var defaultRepo = &Repository{
 }
 
 func Get(bundleName string) (*CrcBundleInfo, error) {
-	return defaultRepo.Use(bundleName)
+	return defaultRepo.Get(bundleName)
+}
+
+func Use(bundleName string) error {
+	_, err := defaultRepo.Use(bundleName)
+	return err
 }
```

Comment:
Might be slightly odd that `Use` does not have the same return values as `(*Repo) Use`

Reply:
Agree, better to have to same return value as `(*Repo) Use` and we can ignore the no used value from the caller function.